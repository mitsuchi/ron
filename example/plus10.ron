op 20 : _ + _
op 10 : _ is _
op 10 : _ = _

a + b = c {
    <c is a + b>
}

main {
    10 + 20 = x
    x + 30 = y
}