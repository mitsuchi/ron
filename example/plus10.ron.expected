x = 30
y = 60
